src/gui_common/ChemicalEquation.cs

```diff
@@ -92,7 +109,6 @@ public override void _Ready()
 
         equationArrowTexture = GD.Load<Texture>("res://assets/textures/gui/bevel/WhiteArrow.png");
 
-        spinner.Visible = showSpinner;
```

Comment:
Why was this removed?

Reply:
It seems now that the method that now sets spinner.Visible isn't called here anymore? Does the process panel display still work?

Reply:
It's now set in `UpdateHeader` which will be called inside the `UpdateEquation` call just below this.

Reply:
Ah okay, I just looked for UpdateHeader call here, didn't consider that another method that's called here would call UpdateHeader.